dashboards-reports/server/routes/utils/constants.ts

```diff
@@ -90,7 +90,7 @@ export const DEFAULT_REPORT_HEADER = '<h1>OpenSearch Dashboards Reports</h1>';
 
 export const SECURITY_CONSTANTS = {
   AUTH_COOKIE_NAME: 'security_authentication',
-  TENANT_LOCAL_STORAGE_KEY: 'opendistro::security::tenant::show_popup',
+  TENANT_LOCAL_STORAGE_KEY: 'opensearch::security::tenant::show_popup',
```

Comment:
the prefix still seems to be opendistro?
https://github.com/opensearch-project/security-dashboards-plugin/blob/main/public/utils/storage-utils.tsx#L18

Reply:
good catch, fixed